Spinner/Spinner.swift

```diff
@@ -84,25 +84,23 @@ public class SpinnerView: NSObject, Spinner {
         let view = showSpinner(inView: button, style: style, color: color)
         
         if let spinnerView = view as? SpinnerView {
-            spinnerView.userInteractionEnabledAtReception = button.isUserInteractionEnabled
```

Comment:
Ah, so that's where the problem came from. I thought I noticed yesterday that the button is still deactivated after the spinner is dismissed, but I didn't have the time to investigate too much. Thanks for solving this :D. We also need to update the Swift 2.3 branch with this fix now